r/src/work.r

```diff
@@ -57,6 +61,28 @@ load_data <- function() {
     return(data)
 }
 
+run_post <- function(req, post_fun) {
+    handle_debug(req)
+    post_fun(req)
+}
+
+handle_debug <- function(req) {
+    task_name <- basename(req$path)
+    is_debug <- debug_step == task_name
+    
+    if (is_debug) {
+        fname <- paste0(task_name, '.RData')
+        fpath_cont <- file.path('/debug', fname)
```

Comment:
Looks like `task_name` will contain slashes that will be interpreted as folder names, won't them?. Is `save()` happy with that?

Reply:
Not sure I understand. The `fpath_cont` will be e.g. `'/debug/${task_name}.RData'` which is used as the file path by `save`

Reply:
You don't understand because I did not make sense :-D I missed the `basename()` and thought that taskname would be something like `/v0/getEmbedding`. Sorry for the noise!

Reply:
Ahhh k np :-D. Always happy to have someone looking out for my bugs :)